features_module_how_to_share_sites_config_with_oth.md

```diff
@@ -93,19 +93,19 @@ Everything we have in our local DB should be in code, so we can share it.
 
 Of course we can't featurize content. Content needs to be [migrated](https://www.drupal.org/project/migrate).
 
-How can we decide which components to pack in one feature? Feature is a set of components for a particular use case, so we need to create feature with a common sense.
+How do we decide which components to pack in one feature? Feature is a set of components for a particular use case, so we need to create Features using common sense.
 
-for example, If we have site with main menu that links to: 
-1. Gallery - made of Content type, Views and Vocabulary (taxonomy)
-2. Blog - made of Content type, Views and Vocabulary (taxonomy)
-3. About page - made of Content type.
+For example, if we have a site with a main menu that links to: 
+1. Gallery - made of content type, Views and Vocabulary (taxonomy)
+2. Blog - made of content type, Views and Vocabulary (taxonomy)
+3. About page - made of content type.
 
 How many features are we going to create from this structure?
```

Comment:
I think this could also use the visual you used in clumping the Features.  Just imagine how overwhelming all these words are.